lua/bufferline/render.lua

```diff
@@ -131,15 +132,17 @@ local function render(update_names)
       icon =
         (letter ~= nil and letter or ' ') ..
         (has_icons and ' ' or '')
-    elseif has_icons then
+    else
       if has_numbers then
-        local number_text = tostring(i)
+        local number_text = tostring(buffer_number)
```

Comment:
And why are we using the buffer number here instead of `i`? It's `i` because the person who requested this feature did it because it makes easier to switch with Alt+[index] (mapped to `BufferGoto [index]`)